lab-alex-bessie-david/lib/transformer.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const transformer = module.exports = function(buff, transformType){
+  if(!Buffer.isBuffer(buff)) throw new Error('not a buffer');
+  var transformTypes = ['blackout', 'invert', 'grayscale', 'invcolors'];
+  if(!transformTypes.includes(transformType)) throw new Error('transform type not supported');
+  
+  switch(transformType){
+  case 'blackout':
```

Comment:
case switches definitely have their place at times, but don't want to be too default in your toolbox -- they are associated with junior devs. Great job getting the transforms to work, though, this is one of the most infamous labs of 401